src/database/queries/proposals/selectWithPagination.sql

```diff
@@ -9,7 +9,7 @@ FROM proposals p
 WHERE
   CASE
     WHEN :search::text != '' THEN
-      pfv.value_search @@ websearch_to_tsquery(:search::text)
+      pfv.value_search @@ websearch_to_tsquery('english', :search::text)
```

Comment:
So glad this was such a simple fix!

Reply:
It's not yet fixed, it has to get merged and deployed...